README.md

```diff
@@ -87,6 +123,14 @@ We document our Coding Guidelines in the [Developer Guide](https://docs.ansible.
 Based on team and community feedback, an initial roadmap will be published for a major or minor version (ex: 2.7, 2.8).
 The [Ansible Roadmap page](https://docs.ansible.com/ansible/devel/roadmap/) details what is planned and how to influence the roadmap.
 
+## FAQ
+How to find a specific module?
```

Comment:
I would not create an FAQ here, just point to the existing one https://docs.ansible.com/ansible/latest/reference_appendices/faq.html

Specially since the questions added here are not that common and you are mixing user and developer ones.